include/cute/atom/copy_traits_xe.hpp

```diff
@@ -1933,4 +1969,54 @@ struct Copy_Traits<XE_1D_STORE_GLOBAL<S, D>> {
     using RefLayout = SrcLayout;
 };
 
+namespace detail
+{
+  template<class PrefetchTileSize, class dtype>
+  auto prefetch_selector(void* ptr = nullptr, int32_t width = 0, int32_t height = 0, int32_t pitch = 0) {
+    if constexpr (get<0>(PrefetchTileSize{}) == 1) {
+      using prefetch_trait = Copy_Traits<XE_2D_U8x1x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<dtype const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2>>{});
+    }
+    if constexpr (get<0>(PrefetchTileSize{}) == 2) {
+      using prefetch_trait = Copy_Traits<XE_2D_U8x2x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<dtype const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2, _2>>{});
+    }
+    if constexpr (get<0>(PrefetchTileSize{}) == 4) {
+      using prefetch_trait = Copy_Traits<XE_2D_U8x4x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<dtype const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2, _4>>{});
+    }
+    if constexpr (get<0>(PrefetchTileSize{}) == 8) {
+      using prefetch_trait = Copy_Traits<XE_2D_U8x8x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<dtype const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2, _8>>{});
+    }
+    if constexpr (get<0>(PrefetchTileSize{}) == 16) {
+      // static_assert(false);
+      using prefetch_trait = Copy_Traits<XE_2D_U8x16x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<dtype const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2, _16>>{});
+    }
+    if constexpr (get<0>(PrefetchTileSize{}) == 32) {
+      using prefetch_trait = Copy_Traits<XE_2D_U8x32x64_LD_N>;
+      using prefetch_atom = Copy_Atom<prefetch_trait, dtype>;
+      return make_tiled_copy(prefetch_atom{}.with(static_cast<char const*>(ptr), width, height, pitch),
+                                           Layout<Shape<_1, _16>>{},
+                                           Layout<Shape<_2, _2, _32>>{});
```

Comment:
Would this still work if the user change the thread layout in the tiledMMA? 

```c++
  using TiledMma = TiledMMA<MMA_Atom<XE_8x16x16_F32BF16BF16F32_TT>,
          Layout<Shape<_8,_4,_1>>,
          Tile<_64,_64,_32>>; 
```

Reply:
Yes, copy traits will be adapted to TiledMMA automatically